README.md

```diff
@@ -7,7 +7,8 @@ Note: It interfaces with WMI on the local machine, therefore it only runs on Win
 
 ---
 
-NOTE: This project is no longer being actively maintained.  If you would like
-to become its new owner, please contact tlimoncelli at stack over flow dot com.
+NOTE: This project is no longer being actively maintained.
+
+We recommend you refer to this fork: https://github.com/yusufpapurcu/wm
```

Comment:
FYI: Missing the "i" in the repo name.

Reply:
Darn cut-and-paste!   :-)

Thanks, I've fixed it.